radlab-ui/automation/terraform/modules/tf-builder/Dockerfile

```diff
@@ -25,6 +25,8 @@ FROM gcr.io/cloud-builders/gcloud
 
 RUN apt-get -y -qq update && \
     apt-get install -y -qq curl && \
+    apt-get install -y -qq python3-pip && \
+    apt-get install -y -qq python3-venv && \
```

Comment:
Any reason to not combine these in one line?

Reply:
Changes will be made in a separate PR based on your feedback.